level1a/handlers/mats_utils/coordinates.py

```diff
@@ -12,6 +12,7 @@
 
 DEGREES_PER_HOUR = 15
 SECONDS_PER_HOUR = 3600
+SECONDS_PER_DEGREE = SECONDS_PER_HOUR / DEGREES_PER_HOUR
 
 PLANETS = load(str(pathlib.Path(__file__).parent.resolve() / "de421.bsp"))
```

Comment:
are we keeping this version of the planets data ?

Reply:
I am merging this with the current version and creating a separate issue instead.